Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -771,6 +780,23 @@ open class TextStorage: NSTextStorage {
         }
     }
 
+    /// Inserts the MoreAttachment at the specified position
+    ///
+    @discardableResult
+    open func replaceRangeWithMoreAttachment(_ range: NSRange) -> MoreAttachment {
+        let message = "MORE"
+        let label = NSLocalizedString("MORE", comment: "Text for the center of the more divider")
+
+        let attachment = MoreAttachment()
+        attachment.message = message
+        attachment.label = NSAttributedString(string: label, attributes: [:])
+
+        let payload = NSAttributedString(attachment: attachment)
```

Comment:
Payload is "the actual information or message in transmitted data".

Can we stick to a more standard `string` name since there's really no transmission involved?